qutebrowser/config/configdata.yml

```diff
@@ -1780,6 +1784,11 @@ tabs.wrap:
   type: Bool
   desc: Wrap when changing tabs.
 
+tabs.tree_tabs:
```

Comment:
This setting needs a change handler to initialize the tree when the setting is turned on. When toggling the setting and restarting, I get a crash in the undo tree (saying node is not defined)